group_utils/javascript/automation/lib/javascript/community/groupUtils.js

```diff
@@ -0,0 +1,347 @@
+/**
+ * Utilities for working with openHAB groups in JavaScript.
+ * This code is not compatible with the GraalVM JavaScript add-on.
+ *
+ * Copyright (c) 2021 Florian Hotze under MIT License
```

Comment:
I don't know if this will cause problems or not. We're small potatoes so it might not matter in the long run but over all I've been using Apache 2.0 for the license for everything in this repo. Would it be possible to use that for consistency? I don't know what it means to mix licenses in one repo like that.

I've been using the following for the copyright/licence at the top of all my files (well, it will be at the top of all my files when I go back and add it, I dropped the ball for awhile). Obviously use your name.

```
/**
Copyright July 23, 2021 Richard Koshak
Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```

Reply:
Yes, no problem to use the Apache 2.0 License. (As ist is very similar to MIT License that I used before.)